writeToStream.js

```diff
@@ -577,4 +588,8 @@ function byteLength (bufOrString) {
   else return Buffer.byteLength(bufOrString)
 }
 
+function isStringOrBuffer (field) {
+  return typeof field === 'string' || Buffer.isBuffer(field)
```

Comment:
you can probably check `field instanceof Buffer` here. That's what `Buffer.isBuffer()`  is doing anyway.

Reply:
Yes, but I actually want to use util.isString to check string type to make it more functional. But deprecated function stops me.
Thanks for all the time reviewing.